pkgs/applications/misc/swappy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, wayland
+, cairo
+, pango
+, gtk
+, pkgconfig
+, cmake
+, scdoc
+, libnotify
+, gio-sharp
+}:
+
+stdenv.mkDerivation rec {
+  name = "swappy-${version}";
+  version = "2020-02-26";
+
+  src = fetchFromGitHub {
+    owner = "jtheoof";
+    repo = "swappy";
+    rev = "b5cc433d75d77759cef139e0e232bde79196f886";
+    sha256 = "08r9hmhzzb5ac4g6zwm7w05n99v0hl2h0w4d8i694hg4pyjxb95y";
+  };
+
+  buildInputs = [ meson ninja pkgconfig cmake scdoc gio-sharp libnotify gtk pango cairo wayland];
+
```

Comment:
As for https://github.com/jtheoof/swappy/issues/10 , try to add (untested):

```
  nativeBuildInputs = [ glibc.dev ];
```

Source: https://linux.die.net/man/3/clock_gettime + `nix-locate unistd.h`.

Reply:
Did not work:

```
../src/buffer.c: In function 'randname':
../src/buffer.c:12:3: error: implicit declaration of function 'clock_gettime' [-Werror=implicit-function-declaration]
   12 |   clock_gettime(CLOCK_REALTIME, &ts);
      |   ^~~~~~~~~~~~~
../src/buffer.c:12:17: error: 'CLOCK_REALTIME' undeclared (first use in this function)
   12 |   clock_gettime(CLOCK_REALTIME, &ts);
      |                 ^~~~~~~~~~~~~~
../src/buffer.c:12:17: note: each undeclared identifier is reported only once for each function it appears in
../src/buffer.c: In function 'create_shm_file':
../src/buffer.c:45:7: error: implicit declaration of function 'ftruncate'; did you mean 'strncat'? [-Werror=implicit-function-declaration]
   45 |   if (ftruncate(fd, size) < 0) {
      |       ^~~~~~~~~
      |       strncat
cc1: all warnings being treated as errors
```

Reply:
This should be fixed now too.